client/spec/components/ProjectDetail.spec.jsx

```diff
@@ -72,9 +73,10 @@ describe( "ProjectDetail Component", () => {
 					owners: [],
 					versions: {}
 				} );
-				component.setProps( {
-					params: { name: "new-project", owner: "new-owner", branch: "branch" }
-				} );
+				ReactDOM.render(
```

Comment:
Hey @elijahmanor, this difference here is that I am re-rending the component, which will call a `componentWillReceiveProps` and so forth, so the reference doesn't change. In fact, overwriting the variable would actually make this test less compelling since I couldn't be sure it updated the existing component vs. creating a new one. Does that make sense?